docs/BesuProxyModel.md

```diff
@@ -1,8 +1,7 @@
 
 **Testing Avalon Proxy model with Hyperledger Besu**
 1.  Set the environment variable ``TCF_HOME`` to the Avalon root directory. Update ``no_proxy`` environment variable if you are behind a proxy
-    server. Add these private IP - ``172.13.0.2, 172.13.0.3, 172.13.0.4, 172.13.0.5``. These are the IP addresses used in the default Besu 
-    network defined in the corresponding [docker-compose file](dev-environments/besu/docker-compose.yaml)
+    server. Add these hostnames - ``node1.avalon.org, rpc.node1.avalon.org, node2.avalon.org, rpc.node2.avalon.org``. These are the IP addresses used in the default Besu network defined in the corresponding [docker-compose file](dev-environments/besu/docker-compose.yaml)
```

Comment:
Your instructions do not show how to discover the IP addresses for 
node1.avalon.org, rpc.node1.avalon.org, node2.avalon.org, rpc.node2.avalon.org.
Before they were IP addresses on the Docker network, 172.13.0.0.  You need to document the commands needed to find out what the IP addresses are or should be.

Reply:
[‎3/‎26/‎2020 10:22 AM]  
Change *.avalon.org to *.avalon.local.  That makes it clear it is a local network.  Also avalon.org is already in use and that may cause confusion.

Reply:
Thanks Dan. Updated.